src/Plugin/TfaBasePlugin.php

```diff
@@ -61,7 +61,8 @@
    *       Plugin-specific context for use during TfaSetup.
    *
    */
-  public function __construct(array $context) {
+  public function __construct(array $context, array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
The pluginDefinitions are setup in PluginBase class which this class extends.
All the plugins extend this class.